src/index.css

```diff
@@ -41,3 +41,8 @@ code {
 	top: 50%;
 	z-index: -1;
 }
+
```

Comment:
we can add custom css to a material UI generated css class by defining it in `createStyles`
like this 

````javascript
const useStyles = makeStyles({
  root: {
    color: 'red',
    '& .MuiTableCell-root.': {
      padding-top: 1px;
	  padding-bottom: 1px;
      '& span': {
        color: 'blue'
      }
    }
  },
});
`

Reply:
Shifted the table css in the table component.